### DISCUSSION TITLE
Consistent six-week display in Calendar component

### DISCUSSION THREAD

**Discussion by misoton665** [mantinedev/mantine#4652] [Q&A] [answered]
I have a question regarding the Calendar component and its display options. Currently, when utilizing Mantine's Calendar component, the number of displayed weeks can fluctuate between five and six weeks based on the specific month being viewed. However, what I'm aiming for is a consistent display of six weeks, regardless of the month.

I've observed that popular calendar applications like Google Calendar, Calendar on macOS, and Cron consistently maintain a fixed display of six weeks in their monthly views. I'm curious if anyone has insights or suggestions on how to implement this unchanging six-week display in the Calendar component.

<img width="205" alt="スクリーンショット 2023-08-14 18 49 11" src="https://github.com/mantinedev/mantine/assets/3793547/262c0d01-1c29-4120-b214-4c6661d3e75d">

**Comment by rtivital** [comment, marked as answer]
Calendar component does not support this feature

**Reply by misoton665** [reply]
@rtivital Thank you for your response.
Before creating a pull request to implement this feature, I would like to confirm if it is suitable for Mantine. The feature I am considering adding is a prop called "minNumberOfWeeks" which allows specifying the minimum number of weeks.

**Reply by rtivital** [reply]
Currently, Mantine is in code freeze – no new features are planned to be added until 7.0 release. You are welcome to submit a PR after v7 is out (in a couple of months).

**Reply by misoton665** [reply]
Understood. I will wait for the release of v7.
